recipe/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+mesa-libGL
```

Comment:
IIUC the new compilers ignore this. Should we add a CDT instead?

Reply:
Feel free to try. I couldn't get the CDT to work without this.

Reply:
incidentally, when I added this line, the CDT was no longer necessary

Reply:
Looks like that broke the Linux builds :-/

Reply:
yeah, i don't know what changed in the anvil image, many GL things seem to be breaking now. I think this really just needs the yum dependency.

Reply:
There is some discussion on gitter about it. I'll check it out tomorrow.
Meanwhile I'm OK just re-adding the yum_requires.